weaviate/values.yaml

```diff
@@ -146,21 +146,35 @@ serviceMonitor:
 startupProbe:
   # For kubernetes versions prior to 1.18 startupProbe is not supported thus can be disabled.
   enabled: false
-
+  probeType: httpGet
+  probe:
+    httpGet:
+      path: /v1/.well-known/ready
+      port: 8080
```

Comment:
I think that it can be made even simpler. I would remove `probeType` and just leave `probe` with an example of a second definition type like so:
```
probe:
  httpGet:
    path: /v1/.well-known/ready
    port: 8080
  # By default Weaviate uses HTTP probe but one wants to configure a command probe then
  # the following part should be uncommented
  # exec:
  #    command: ["/bin/sh", "-c", "wget --spider --server-response --tries=1 --timeout=30 -o /dev/null localhost:8080/v1/.well-known/live"
```

That way a user would know how he can change this to a command probe.